app/controllers/manage/questions_controller.rb

```diff
@@ -12,16 +12,23 @@ def create
   def filter
     @questions = Question.search_question(params)
     @questions = @questions.paginate(:page => params[:page] || 1, :per_page => 15)
-    render :index
+    if params[:from] == "question_crud"
+      render '/manage/admins/question_crud'
+    else
+      render :index
+    end
   end
 
   def disapprove
     @question.disapprove(params[:disapproval_reason], current_user)
+    flash[:notice] = "#{@question.id} was disapproved"
     redirect_to '/manage/admins/question_crud'
```

Comment:
i'm using the rails 'flash' feature to flash a message to the user when something is updated (try it out by trashing/approving/disapproving a question)